docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -128,9 +128,9 @@ const ItemButton = styled(Item, {
       }
       return theme.palette.text.primary;
     })(),
-    fontSize: depth === 1 ? theme.typography.pxToRem(12) : theme.typography.pxToRem(14.5),
-    fontWeight: depth === 1 ? 700 : 500,
-    margin: theme.spacing(0.5, 0),
+    fontSize: theme.typography.pxToRem(depth === 0 ? 14.5 : 12),
+    fontWeight: depth === 0 ? 600 : 700,
```

Comment:
I thought we said we wouldn't use 600? https://github.com/mui-org/material-ui/issues/28327

Reply:
Oh yeah, great catch. We aren't importing the 600 font-weight, only 400, 500, and 700. 600 ends up rendering the 700.